### PR TITLE
Add News Item Gene Panel Patient View

### PR DESCRIPTION
**Pull request by inodb**
See rich diff: https://github.com/cBioPortal/cbioportal/pull/6901/files?short_path=6f95322#diff-6f953229832059bab3fe229d4af08b52

### CODE REVIEW

File: docs/News.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+## Dec 10, 2019
+* **New Feature**: Show gene panel information on the _Patient View_ page. This helps to distinguish between whether a mutation in a particular gene was not present in the sample or not profiled ([example](https://www.cbioportal.org/patient?caseId=P-0001613&studyId=lung_msk_2017)):
```
Comment (jjgao) [inline docs/News.md]: Wondering if we can find an example with a mutation in one example but not sequenced in another?

Comment (inodb) [reply docs/News.md]: that is in this one right or did I misunderstand? By default it's filtered:

Screen Shot 2019-12-10 at 7 14 44 PM

Comment (inodb) [reply docs/News.md]: it might make sense to show mutations profiled in any sample by default?

Comment (tmazor) [reply docs/News.md]: I definitely agree we should show all mutations by default - I actually made an issue about this the other day :) (#6890 )  If possible, would be great to change this before we make the announcement.

Also how about this case as an example: https://www.cbioportal.org/patient?caseId=P-0002208&studyId=msk_impact_2017
The gene that isn't on one of the panels has a likely oncogenic mutation which I think makes for a nice example.

Comment (tmazor) [reply docs/News.md]: For what it's worth, I also found another issue (#6891) - not sure if it's practical to fix that before this news announcement, but if possible that would be great.

Comment (inodb) [reply docs/News.md]: Thanks Tali! I like that example as well. I think we can wait with the announcement until we make those fixes. @jjgao do u agree to change the default behavior?

Comment (jjgao) [reply docs/News.md]: I agree with @tali.

**Comment by inodb** [comment]
Added here: https://github.com/cBioPortal/cbioportal/pull/6987

**Pull request closed without merging**